### ISSUE TITLE
[rx-51] list-operators fails, even when networks are available

### ISSUE THREAD

**Issue by pavelmachek** [postmarketOS/pmbootstrap#711]
list-operators returns empty list, even with modem connected to the network:

```
localhost:/my/ofono/test$ sudo python2 ./list-operators 
[ /n900_2 ]

```

This is probably regression in ofonod; old ofonod version in debian works on N900.

**Comment by craftyguy** [comment]
I recently updated ofono to 1.21 (was 1.20 previously). Did 1.20 work for you? I don't really use the ofono test scripts any more, at least not for managing data connections.

**Comment by pavelmachek** [comment]
No, unfortunately. The version where list-operators works for me is 1.6.

**Comment by ollieparanoid** [comment]
In case you don't have any better ideas, you could bisect it (with release versions, e.g. does it work in 1.13? then in 1.16 or 1.10?, ...).
